src/constants.rs

```diff
@@ -9,6 +9,9 @@ pub const ANNOTATION_HOPRD_NETWORK_REGISTRY: &str = "hoprds.hoprnet.org/network_
 pub const ANNOTATION_HOPRD_FUNDED: &str = "hoprds.hoprnet.org/funded";
 pub const ANNOTATION_HOPRD_LOCKED_BY: &str = "hoprds.hoprnet.org/locked_by";
 pub const ANNOTATION_REPLICATOR_NAMESPACES: &str = "replicator.v1.mittwald.de/replicate-to";
+pub const ANNOTATION_INGRESS_PROXY_CONNECT_TIMEOUT: &str = "nginx.ingress.kubernetes.io/proxy-connect-timeout";
+pub const ANNOTATION_INGRESS_PROXY_READ_TIMEOUT: &str = "nginx.ingress.kubernetes.io/proxy-read-timeout";
+pub const ANNOTATION_INGRESS_PROXY_SEND_TIMEOUT: &str = "nginx.ingress.kubernetes.io/proxy-send-timeout";
```

Comment:
This would preferably be located in the values.yaml.

Reasoning: What if you want to use a different ingress like traefik? You want to be able to override these implementation specific annotations.

Reply:
Solved